config/sync/core.date_format.cyber_security_medium.yml

```diff
@@ -0,0 +1,8 @@
+uuid: e4592b0d-44b5-49bf-9f49-9435bef98bf2
+langcode: en
+status: true
+dependencies: {  }
+id: cyber_security_medium
+label: 'Cyber security medium'
+locked: false
+pattern: m/d/Y
```

Comment:
As we are aiming for a UK audience, wouldn't d/m/Y be better here ?

Reply:
Yes that would actually be a good idea, I've made the required change and committed.